Add API delays and Hungarian translation

Dwokfur opened the pull request:
Introduce delays due to API rate restriction and add Hungarian translation

Dwokfur committed:
Increase sleep duration in climate.py to 30 seconds

Dwokfur committed:
Increase sleep duration in cover.py to 30 seconds

Dwokfur committed:
Add 30 seconds sleep in select.py for API calls

Dwokfur committed:
Introduce delays in sensor API calls

Added sleep delays to API calls for rate limiting.

Dwokfur committed:
Introduce delays in API calls for Liebherr switches

Added delays before API calls to manage request timing.

Dwokfur committed:
Stub file for Hungarian translation

Dwokfur committed:
Update Hungarian translations in hu.json

Updated Hungarian translations for notifications and configuration steps.

Dwokfur committed:
Fix capitalization in door alarm translation

Ayuus commented:
Syntax error in sensor.py, line 175, I think.

Dwokfur commented:
> Syntax error in sensor.py, line 175, I think.

Ouch, there's a g left at the end of the line.

Dwokfur committed:
Fix typo in async_update method

Dwokfur commented:
https://github.com/Dwokfur/liebherr/commit/c840cf52f56b3984e985d62fd6f696092513db55

Ayuus commented:
IS this PR supposed to work after implementing PR 34 (AutoDoorControl)?

Dwokfur commented:
> IS this PR supposed to work after implementing PR 34 (AutoDoorControl)?

I've fixed the problem on my physical HA install and wanted to propose a solution, so I forked the main branch and created the pull request to have a discussion. Therefore I suspect it won't contain the enhancements of PR 34, nor does it have icemaker related stuff like in https://github.com/skatsavos/liebherr.

Robbe-B commented:
@bhuebschen, 
Can we merge & release this already to make the integration working again ? 
Kind regards

Ayuus commented:
> @bhuebschen, Can we merge & release this already to make the integration working again ? Kind regards

I would prefer having #34 merged first, as that one adds AutoDoorControl and IceMaker.

Dwokfur commented:
> > @bhuebschen, Can we merge & release this already to make the integration working again ? Kind regards
> 
> I would prefer having #34 merged first, as that one adds AutoDoorControl and IceMaker.

There are two sleep(3)s and a sleep(5) in that pull request, which I would also tend to increase to 30...

Ayuus commented:
> > > @bhuebschen, Can we merge & release this already to make the integration working again ? Kind regards
> > 
> > 
> > I would prefer having #34 merged first, as that one adds AutoDoorControl and IceMaker.
> 
> There are two sleep(3)s and a sleep(5) in that pull request, which I would also tend to increase to 30...

I will be happy to test today:-)

Dwokfur committed:
Decrease sleep duration in async functions

Reduced sleep duration from 30 seconds to 5 seconds in multiple async functions to improve responsiveness.

Dwokfur committed:
Update cover.py

Dwokfur committed:
Decrease sleep time in select.py

Reduced sleep duration from 30 seconds to 5 seconds in multiple locations.

Dwokfur committed:
Remove redundant asyncio.sleep calls

Removed unnecessary sleep calls in sensor setup and update methods.

Dwokfur committed:
Remove redundant asyncio.sleep calls

Removed unnecessary sleep calls to improve performance.

Dwokfur commented:
> > > > @bhuebschen, Can we merge & release this already to make the integration working again ? Kind regards
> > > 
> > > 
> > > I would prefer having #34 merged first, as that one adds AutoDoorControl and IceMaker.
> > 
> > 
> > There are two sleep(3)s and a sleep(5) in that pull request, which I would also tend to increase to 30...
> 
> I will be happy to test today:-)

Re-created my fork by first including skatsavos's icemaker/autodoor related changes and reproduced an updated pull request.
